RogueTech Core/GameTips/general.txt

```diff
@@ -1,23 +1,23 @@
 If your 'Mech is considered invalid, you can hit the "Confirm" button to see what issues it has.
 Switch between your equipped ammunition types by clicking on the name of your current ammunition in the 'Mech's weapon panel.
-Vehicles take each point of heat damage as two points of normal damage.
-Every Important Information on item is either in the Traits or Brightly Coloured.
-Through Armor Criticals are defined by Weapons, a Gauss slug can punch through the thickest hides but a Laser only melts the surface.
-Putting your cursor over you weapons will show you exactly why you cant hit the flanking stealthed light hiding on a forest hill.
-Can't Headshot a moving target with Offensive Push? Try to find a Headshot Capable FCS
+Vehicles, turrets, and buildings take each point of heat damage as two points of normal damage.
+All the important Information on an item is either in the "Traits" section or otherwise brightly coloured.
+Through-Armor-Critical (TAC) chance is defined by weapon; a gauss slug can punch through the thickest hides but a laser only melts the surface.
+Hovering your cursor over your weapon hit percentage will show you exactly why you cant hit the flanking stealthed light hiding on a forest hill.
+Headshots are disabled in Offensive Push unless you have a FCS or BC that enables them.
 Stealth Armor cannot be Multi-Targeted
 Evasion pips don't go away when you shoot at a target. 
 TAB can be used to toggle between Targets.
-Press V to cycle between available melee Targets.
-Mouseover the Cybereye in your mech's status panel to see your sensor rolls.
+Press V to cycle between available melee targets.
+Mouseover the cybereye in your mech's status panel to see your sensor rolls.
 The colored dots on the ground show different types of terrain.
-Ctrl+click on Ammunition counter to eject Ammo before Moving.
-CTRL-A toggles visibility of ECM bubbles.
-Shift-Click on the skills tab of a pilot in the Barracks to reset their skills; Requires Argo Training Module Upgrade.
+CTRL+Click lets you jettison ammo bins to avoid heat-related explosions!
+CTRL+A toggles visibility of ECM bubbles.
+If you have the Argo Training Module Upgradem you can Shift+Click on the skills tab of a pilot in the Barracks to reset their skills.
```

Comment:
This requires the second training module upgrade, right?

```suggestion
If you have the second Argo Training Module Upgrade you can Shift+Click on the skills tab of a pilot in the Barracks to reset their skills.
```
At the very least, there's an `m` on Upgrade.

Reply:
i can type don't bully me :<